docker-compose.override.local-mac.yml

```diff
@@ -0,0 +1,6 @@
+version: '3'
+services:
+  celery:
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - /usr/local/bin/docker:/usr/bin/docker
```

Comment:
Would it be better to just change `docker-compose.override.yml` to have
```
 volumes:
      - ${DOCKER_SOCK:-/var/run/docker.sock}:/var/run/docker.sock
      - ${DOCKER_CMD:-/usr/bin/docker}:/usr/bin/docker
```
and then you would override this with environment variables?

Reply:
Much simpler